pkg/controller/controllercmd/builder.go

```diff
@@ -304,6 +314,17 @@ func (b *ControllerBuilder) Run(ctx context.Context, config *unstructured.Unstru
 		return nil
 	}
 
+	if b.canUseSNOLeaderElection {
+		infraStatus, err := clusterstatus.GetClusterInfraStatus(ctx, clientConfig)
+		if err != nil || infraStatus == nil {
+			eventRecorder.Warningf("ClusterInfrastructureStatus", "Failed to get cluster infrastructure status because of %s", err)
+			klog.Warningf("unable to get cluster infrastructure status for leader election (falling back to default): %v", err)
```

Comment:
event and warning should match and I think

`klog.Warningf("unable to get cluster infrastructure status, using HA cluster values for leader election: %v", err)`

Reply:
I like that message too, updated logging and event messages to match